pr_6/javascript/random_samples/direct-response-write.js

```
@@ -0,0 +1,184 @@
+app.get('/', function (req, res) {
+    var user = req.query.name;
+
+    msg = "Hi " + user
+    // ruleid: direct-response-write
+    res.send('Response</br>' + msg);
+});
+// {/fact}
+
+// {fact rule=cross-site-scripting@v1.0 defects=1}
+var msg = '';
+app.get('/3', function (req, res) {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)